apps/web/src/pages/Users/UserInformationPage/components/UserCandidatesTable/UserCandidatesTable.tsx

```diff
@@ -0,0 +1,261 @@
+import tableMessages from "~/components/PoolCandidatesTable/tableMessages";
+import useRoutes from "~/hooks/useRoutes";
+import { PoolCandidate_BookmarkFragment } from "~/components/CandidateBookmark/CandidateBookmark";
+import { getFullPoolTitleLabel } from "~/utils/poolUtils";
+import processMessages from "~/messages/processMessages";
+import adminMessages from "~/messages/adminMessages";
+import {
+  JobPlacementDialog_Fragment,
+  jobPlacementDialogAccessor,
+} from "~/components/PoolCandidatesTable/JobPlacementDialog";
+import cells from "~/components/Table/cells";
+import accessors from "~/components/Table/accessors";
+
+import {
+  bookmarkCell,
+  bookmarkHeader,
+  candidacyStatusAccessor,
+  candidateNameCell,
+  currentLocationAccessor,
+  finalDecisionCell,
+  notesCell,
+  priorityCell,
+  processCell,
+} from "./helpers";
+
+type PoolCandidateSlice = Pick<
+  PoolCandidate,
+  | "id"
+  | "user"
+  | "notes"
+  | "submittedAt"
+  | "status"
+  | "pool"
+  | "placedDepartment"
+  | "suspendedAt"
+  | "isBookmarked"
+>;
+
+const columnHelper = createColumnHelper<PoolCandidateSlice>();
+
+export interface UserCandidatesTableProps {
+  user: UserInfoFragment;
+  departments: Department[];
+  title: string;
+}
+
+const UserCandidatesTable = ({
+  user,
+  departments,
+  title,
+}: UserCandidatesTableProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const poolCandidatesUnpacked = user.poolCandidates?.filter(notEmpty) ?? [];
+  const poolCandidatesMappedToType: PoolCandidateSlice[] =
+    poolCandidatesUnpacked.map((candidate) => {
+      return {
+        id: candidate.id,
+        user: candidate.user,
+        notes: candidate.notes,
+        submittedAt: candidate.submittedAt,
+        status: candidate.status,
+        pool: candidate.pool,
+        placedDepartment: candidate.placedDepartment,
+        suspendedAt: candidate.suspendedAt,
+        isBookmarked: candidate.isBookmarked,
+      };
+    });
+  const candidateIds = poolCandidatesMappedToType.map(
+    (candidate) => candidate.id,
+  );
+
+  const columns = [
+    columnHelper.display({
+      id: "isBookmarked",
+      header: () => bookmarkHeader(intl),
+      enableHiding: false,
+      cell: ({ row: { original: poolCandidate } }) =>
+        bookmarkCell(
+          poolCandidate as FragmentType<typeof PoolCandidate_BookmarkFragment>,
+          poolCandidate.isBookmarked,
+        ),
+      meta: {
+        shrink: true,
+        hideMobileHeader: true,
+      },
+    }),
+    columnHelper.accessor(
+      () => getFullNameLabel(user.firstName, user.lastName, intl),
+      {
+        id: "candidateName",
+        header: intl.formatMessage(tableMessages.candidateName),
+        sortingFn: normalizedText,
+        cell: ({ row: { original: poolCandidate } }) =>
+          candidateNameCell(user, poolCandidate, paths, intl, candidateIds),
+        meta: {
+          isRowTitle: true,
+        },
+      },
+    ),
```

Comment:
I think this column is meant to be removed.

https://github.com/GCTC-NTGC/gc-digital-talent/pull/10342#issuecomment-2105265420

Reply:
Um, I think it was to be hidden by default?

Reply:
Yeah I agree with @esizer , but  I don't know what's best in terms of implementation, I just assumed that reusing a table and hiding anything that's not valuable would be the easiest solution. 

But I agree that information that repeats elsewhere is not necessary on the table itself. Mostly the table only needs to show information regarding the user/pool relationship like the final decision, status, expiry date, placement etc.

Reply:
Okay, fair. I removed some accessors. https://github.com/GCTC-NTGC/gc-digital-talent/pull/10342/commits/4218e22d60db862c7c4ba57d704e20a8aa81d9e5
Am keeping in the name one as that is more than just the name but also how you get to the specific application. It is the same way as for the pool candidates table and elected to have it work and appear the same.